build.gradle

```diff
@@ -180,6 +181,7 @@ dependencies {
     api 'com.exactpro.th2:grpc-common:4.2.0-dev'
     api "com.exactpro.th2:cradle-core:${cradleVersion}"
     api 'io.github.microutils:kotlin-logging:2.1.21'
+    api 'io.netty:netty-buffer'
```

Comment:
I think we should add a `because` description here. Because this is the first candidate to move into th2-protocol lib when we separate it from common lib